File: WORDCNT.py
numTests = int(raw_input())
for i in range(numTests):
	prev = 0
	maximum = count = 1
	data = str(raw_input())
	strings = data.split(' ')
	for s in strings:
		if len(s)>0:
			if prev == len(s):
				count+=1;
				if count > maximum:
					maximum = count
			else:
				count = 1;
				prev = len(s)
	print maximum				

File: WORDCNT.cpp
#include<stdio.h>
#include<iostream>

#define MAXLEN 20010
using namespace std;
int main()
{
	int c,count,max,prev,justStarted,wordLen,i;
	static int numTests;
	
	scanf("%d",&numTests);
	fgetc(stdin);
	for(i=0;i<numTests;i++)
	{
		wordLen = 0;
		prev = 0;
		max = 1;
		justStarted = 1;
		count = 1;
		while(1)
		{
			c = fgetc(stdin);
			if(c!=' ' && c!='\n')
			{
				wordLen++;
			}
			else
			{
				//printf("%d\n",wordLen);
				if(wordLen == prev)
					count++;
				else
				{
					count = 1;
				}
				if(count > max)
					max = count;
				prev = wordLen;
				wordLen = 0;
			}
			if(c == '\n')
				break;
		}
		printf("%d\n",max);
	}
}
